DeAceroDLabs/marketplace-frontend
# [DMP-630] Added fallback image to carousel

## What is done

[[DMP-630] Add Fallback image to Carousel](https://deacero.atlassian.net/browse/DMP-630)

## How is done

1. On the Carousel component added a checkpoint to verify that there is an image to display.
2. Display the fallback image, similar to the products' cards, if there isn't an image.

## How to test

1. Run the project via `npm start`
3. On the catalog section click on any product.
4. You will be redirected to the product page.
5. Wait for the request to finish loading. 
6. You should see the fallback image

## Evidence 
Before
![Screen Shot 2022-10-07 at 0 04 43](https://user-images.githubusercontent.com/68872548/194473815-88d9e6c5-65b4-4f8b-827f-8a1bc4109a07.png)

After
![Screen Shot 2022-10-07 at 0 25 41](https://user-images.githubusercontent.com/68872548/194473906-e9bf2cc9-639f-4056-8bf8-c4bd666c526b.png)
